### DISCUSSION TITLE
Radio ID Range

### DISCUSSION THREAD

**Discussion by robertlynch3** [chuot/rdio-scanner#171] [Ideas]
My county uses radio ID ranges based on towns, so 1000-1999 is town A, 2000-2999 is town B. It would be a nice feature to be able to setup a range an have an 'catch-all' label for them. If an ID has a unit set already, that should take precedence, but if not, use the label from the range.

Even further, maybe some sort of Regex, so if I say 1000-1999 is town A, I can say grep everything after 1 and assign that to the radio number so 1005 can be labeled Town A radio 5.

**Comment by chuot** [comment]
It's a good idea. Please open an issue for this proposal. I will try to do it for v6.3.

**Comment by robertlynch3** [comment]
#173 opened.
